src/routes/Profile/ProfilePageWalletConnected/ProfilePageWalletConnected.stories.tsx

```diff
@@ -40,6 +42,21 @@ const Template: ComponentStory<typeof ProfilePageWalletConnected> = (args) => {
     setClaimButtonBulkMode('claim');
   }, []);
 
+  // fake copying
+  function handleOnCopyLinkButtonClick() {
+    setCopyLinkButtonMode('copying');
+    try {
+      setTimeout(() => {
+        setCopyLinkButtonMode('copied');
+        setTimeout(() => {
+          setCopyLinkButtonMode('copy');
+        }, 1500);
+      }, 3000);
```

Comment:
this feels a bit too long

Reply:
this is mocking code of a simulated experience. Not really relevant for the purpose of this PR

Reply:
When I click on the button it feels like it's following this logic even though this is in stories. Is it because of the `doNothing` thing?

Reply:
this is both in stories and in main component. Just so that stakeholders get the feel of the UI will be. We dont publish storybook(yet) so there is no way for them to see the proper UI.